README.md

```diff
@@ -1,26 +1,50 @@
-
 Node.js
-=====
+=======
 
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/nodejs/node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This repository began as a GitHub fork of
-[joyent/node](https://github.com/joyent/node).
-
-Node.js contributions, releases, and contributorship are under an
+The Node.js project is supported by the
+[Node.js Foundation](https://nodejs.org/en/foundation/). Contributions,
+policies and releases are managed under an
 [open governance model](./GOVERNANCE.md).
-We intend to land, with increasing regularity, releases which are
-compatible with the npm ecosystem that has been built to date for
-Node.js.
+
+## Release Types
+
+The Node.js project maintains multiple types of releases:
+
+* **Stable**: planned and signed releases. Released from active development
+  branches of this repository and versioned by [SemVer](http://semver.org/).
+  Code for Stable releases are organized in this repository by major version
+  number, e.g. [v4.x](https://github.com/nodejs/node/tree/v4.x).
+  The major version number of Stable releases will increment every 6 months
+  allowing for breaking changes to be introduced, this happens in April and
+  October every year. Stable release lines beginning in October each year have
+  a maximum support life of 8 months. Stable release lines beginning in April
+  each year will convert to LTS (see below) after 6 months and receive further
+  support for 30 months.
+* **LTS**: Every second Stable release line (major version) will become LTS
+  lines and receive 18 months of _Active LTS_ support and a further 12
+  months of _Maintenance_. LTS release lines are given alphabetically
+  ordered codenames, begining with v4 "Argon". LTS releases are less frequent
+  and will attempt to maintain consistent major and minor version numbers,
+  only incrementing patch version numbers. i.e. no breaking changes or
```

Comment:
The `i.e.` sentence isn't a full sentence. Avoid `i.e.` anyway because too many people think it means "for example". I would just change the whole sentence to `There will be no breaking changes or feature additions.`